Lib/test/_test_multiprocessing.py

```diff
@@ -1039,9 +1039,10 @@ def test_timeout(self):
         start = time.time()
         self.assertRaises(pyqueue.Empty, q.get, True, 0.200)
         delta = time.time() - start
-        # Tolerate a delta of 50 ms because of the bad clock resolution on
-        # Windows (usually 15.6 ms)
-        self.assertGreaterEqual(delta, 0.150)
+        # bpo-30317: Tolerate a delta of 100 ms because of the bad clock
+        # resolution on Windows (usually 15.6 ms). x86 Windows7 3.x once
+        # failed because the delta was only 135.8 ms.
+        self.assertGreaterEqual(delta, 0.100)
```

Comment:
Could we use the more permissive delta only when this test is being executed on Windows? It complicates the test a bit (two different values for the check) but it will not be permissive in all platforms, only in the ones with a bad clock resolution (Windows). What do you think?